MarkiyanAndrusevych/HW5/task5.py

```diff
@@ -0,0 +1,22 @@
+# #first task of HW5
+# input_string = input("Enter a list of integers using spaces :) ")
+# int_list = [int(x) for x in input_string.split()]
+# float_list= [float(element) for element in int_list]
+# print("Your integer list:",int_list)
+# print("Converted to float:", float_list )
+
+# #second task of HW5
+# n = int(input("Enter the number: "))
+# fib = [0,1]
+# while fib[-1] + fib[-2] <=n:
+#  new_fib = fib[-1] + fib[-2]
+#  fib.append(new_fib)
+#
+# print("Ur Fibonacci numbers: ", fib)
+
+# #third task of HW5
```

Comment:
good job
look at this implementation
input_number = int(input("Input natural number: "))

fact = 1

if input_number == 1 or input_number == 0:
    print("Factorial {}! =".format(input_number), fact)
elif input_number < 0:
    print("Factorial non positive number don't exist")
else:
    for item in range(1,input_number + 1):
        fact *= item
    print("Factorial {}! =".format(input_number), fact)